packages/migration/src/mods/7.0.0/createRefFromReact.ts

```diff
@@ -0,0 +1,40 @@
+import { mod } from 'riceburn';
+import ts from 'typescript';
+import { migration, IMigrationOptions, ModResult } from '../../migration';
+
+const commaBeforeRegex = new RegExp('\s?,\s*createRef');
+const commaAfterRegex = new RegExp('createRef\s*,\s?');
+const noCommaRegex = new RegExp('createRef');
+
+export default migration(
+  'createRef should come from React.createRef, not from office-ui-fabric-react',
+  (opts: IMigrationOptions): ModResult[] => {
+    return mod('**/*.ts?(x)', opts).asTypescript((node, modder) => {
+      if (ts.isImportDeclaration(node)) {
+        if (node.moduleSpecifier.getText() === "'office-ui-fabric-react'") {
+          if (node.importClause && node.importClause.namedBindings) {
+            const namedBindings = node.importClause.namedBindings;
+            if (ts.isNamedImports(namedBindings)) {
+              let foundCreateRef = false;
+              namedBindings.forEachChild(c => {
+                if (c.getText() === 'createRef') {
+                  foundCreateRef = true;
+                }
+              });
+              if (foundCreateRef) {
+                const importText = node.getFullText();
```

Comment:
Frustrating not to be able to just rebuild this node from types and output it. This model seems pretty fragile.

Reply:
You mean to create a prettier'ified TS code based on the AST collected here? We can do this, but I've found the other tools are NOT typed like typescript itself, so that sucks. But we can definitely construct code from nodes if needed and then run prettier on it?